content/en/blog/releases/v0.7.0-release-notes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Build v0.7.0"
+date: 2021-12-17T10:29:59-04:00
+draft: false
+---
+
+Shipwright Build v0.7.0 can be found on GitHub at [shipwright-io/build v0.7.0](https://github.com/shipwright-io/build/releases/tag/v0.7.0).
```

Comment:
Can you also please add content here for the `shp` command line release, along with cli install instructions?

Reply:
Done.